driver/ssh.go

```diff
@@ -36,33 +37,37 @@ func (d *SSH) String() string {
 
 // set the goph Client
 func (d *SSH) Client() (*goph.Client, error) {
-	var err error
-	var client *goph.Client
-	var callback ssh.HostKeyCallback
-	if d.Port != 0 {
-		port = d.Port
-	}
-	auth, err := goph.Key(d.KeyFile, d.KeyPass)
-	if err != nil {
-		return nil, err
-	}
-	if d.CheckKnownHosts {
-		callback, err = goph.DefaultKnownHosts()
+	if d.SessionClient == nil {
+		var err error
+		var client *goph.Client
+		var callback ssh.HostKeyCallback
+		if d.Port != 0 {
+			port = d.Port
+		}
```

Comment:
Default value if not specified is 22. Is this used if it's not specified?

Reply:
It is already specified as var on the entire file